docs/source/models/collisional_ionization.rst

```diff
@@ -41,6 +52,16 @@ We therefore implemented additional conditions to mediate unphysical behavior bu
     It is strongly suggested to do approximations for **every** setup or material first.
     To that end, a parameter scan with [FLYCHK]_ can help in choosing a reasonable value.
 
+3. Low electron-temperature cutoff
+
+    Depending on the material the Thomas-Fermi prediction for the average charge
```

Comment:
In sphinx as in latex, please write one sentence per line independent of the line length.